src/ray/common/ray_config_def.h

```diff
@@ -274,6 +274,10 @@ RAY_CONFIG(int32_t, minimum_gcs_reconnect_interval_milliseconds, 5000)
 /// Whether start the Plasma Store as a Raylet thread.
 RAY_CONFIG(bool, plasma_store_as_thread, false)
 
+/// Whether to release worker CPUs during plasma fetches.
+/// See https://github.com/ray-project/ray/issues/12912 for further discussion.
+RAY_CONFIG(bool, release_resources_during_plasma_fetch, false)
```

Comment:
Do we still need this flag? I can't really think of any cases where we would want this to be true.

Reply:
It's here just in case we want to easily disable it (feature flag).